test/reference32/typical_merge_leadfollow.txt

```diff
@@ -33,127 +33,127 @@ quote
     phase_stop = min(y_lvl.shape, y_lvl_i1, x_lvl_i1)
     if phase_stop >= 1
         i = 1
+        if y_lvl.idx[y_lvl_q] < 1
```

Comment:
jumpers don't need to emit a seek before the loop, since everyone in a jumper will seek at the start of each block

Reply:
Great point -- I updated a code to remove a seek inside the jumper's loop.